modules/aws_k8s_base/aws-k8s-base.json

```diff
@@ -61,6 +61,21 @@
       "description": "Expose self-signed SSL certs.",
       "default": false
     },
+    "cert_manager_values": {
+      "type": "object",
+      "description": "Certificate Manager helm chart additional values. [Available options](https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values)",
+      "default": {}
+    },
+    "linkerd_values": {
+      "type": "object",
+      "description": "Linkerd helm chart additional values. [Available options](https://artifacthub.io/packages/helm/linkerd2/linkerd2/2.10.2?modal=values)",
+      "default": {}
```

Comment:
the default is actually what we have here 
https://github.com/run-x/opta/blob/main/modules/aws_k8s_base/tf_module/values-ha.yaml
we can document this - so it's clear to the user where this values are coming from

Reply:
The defaults for High availability we already have, but we are just appending any more that the user would want.